documentation/api/index.html

```diff
@@ -88,22 +88,45 @@
         instance (both locally and on AWS), Flyway will automatically use the JDBC url, user and password
         <a href="https://boxfuse.com/docs/databases#envvars">provided by Boxfuse</a>.</div>
 
-    <h2>Programmatic Configuration (Android)</h2>
+    <h2 id="android">Android Programmatic Configuration (Android Studio)</h2>
 
```

Comment:
Not really. But Android development these days is almost almost documented in terms of Android Studio. Technically, the description only relies on Gradle (because of the definition of deps). But there is an old Gradle-way and a new Gradle-way (which came with Android Studio). I think the need to define assets is different in the old way.

In short: the details in this description are necessary and sufficient for up-to-date newly created android projects, but details are different for older projects.